### PR TITLE
Fix xcode12 warnings

### LINKED ISSUE #303: Fix XCode 12 warnings
Some features have been deprecated.

### PR DESCRIPTION
**Pull request by MykytaKiriushyn**
- Fixes #303

### CODE REVIEW

File: o-fish-ios/Views/Components/Popover/PopoverManager.swift [hunk 1]
```diff
@@ -80,3 +80,9 @@ class PopoverManager {
 //        )
 //    }
 //}
+
+struct PopoverManager_Previews: PreviewProvider {
+    static var previews: some View {
```
Comment (akaktus) [inline o-fish-ios/Views/Components/Popover/PopoverManager.swift:85]: It's incorrect preview

**Review by akaktus — CHANGES_REQUESTED** [review]

File: o-fish-ios/Views/Components/Popover/PopoverManager.swift (hunk 1 above)
Comment (MykytaKiriushyn) [reply o-fish-ios/Views/Components/Popover/PopoverManager.swift:85]: That was added automatically, so I don't think it's even needed

File: o-fish-ios/Views/Components/KeyboardControlling/KeyboardController.swift [hunk 2]
```diff
@@ -17,7 +17,6 @@ final class KeyboardController: ObservableObject {
     }
 
     deinit {
```
Comment (akaktus) [inline o-fish-ios/Views/Components/KeyboardControlling/KeyboardController.swift:19]: remove deinit at all.

**Comment by o-fish-wildaid** [comment]
[Review PRs](https://trello.com/c/LIywX167/325-review-prs)

**Review by am-MongoDB — APPROVED** [review]

**Pull request merged**